src/libopensc/card-epass2003.c

```diff
@@ -39,9 +39,10 @@
 #include <stdlib.h>
 #include <string.h>
 
+#include <openssl/cmac.h>
```

Comment:
this include does not seem to be needed

Reply:
We need this #include <openssl/cmac.h> for function aes128_encrypt_cmac() - epass2003 uses CMAC-AES-128 for SM in FIPS certified mode. (I changed the order of includes in this file, as git-clang-format had a problem with another order.)

Reply:
Ops, incorrect "include"... The PR is already fixed, modified, the unnecessary #include <openssl/evp.h>" is removed.